src/shell.ts

```diff
@@ -55,6 +57,13 @@ function combinePath(basePath : string, relativePath : string) {
     return basePath + separator + relativePath;
 }
 
+function fileUri(filePath : string) : vscode.Uri {
+    if (isWindows()) {
+        return vscode.Uri.parse('file:///' + filePath.replace(/\\/g, '/'));
```

Comment:
perhaps we can use node's path module to make this work?

Reply:
Hmm, I don't see anything in the path module that helps with the `file:` side of things -- am I missing something?  We could probably use it to avoid the backslash replacement but I'm not sure there's much value in that -- what do you reckon?